### PR TITLE
[fix] location of bower components in README.md

### PR DESCRIPTION
**Pull request by KalisaFalzone**

### CODE REVIEW

**Comment by thewarpaint** [comment]
You probably also want to update https://github.com/wallin/angular-gravatar/blob/master/.gitignore#L2 and https://github.com/wallin/angular-gravatar/blob/master/.bowerrc#L2.

**Comment by KalisaFalzone** [comment]
Updated three files. Good catch!

**Comment by thewarpaint** [comment]
Happy to help. I think this one should also be updated: https://github.com/wallin/angular-gravatar/blob/master/karma.conf.js#L12

File: README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Usage
 Include angular-gravatar.js in your application.
 
 ```html
-<script src="components/angular-gravatar/build/angular-gravatar.js"></script>
+<script src="./bower_components/angular-gravatar/build/angular-gravatar.js"></script>
```
Comment (thewarpaint) [inline README.md]: Not sure if the `./` is necessary here.

**Comment by KalisaFalzone** [comment]
Removed the ./ and updated the karma file. :)

**Comment by KalisaFalzone** [comment]
:)

**Pull request merged**